provider/source/src/time_zones/convert.rs

```diff
@@ -106,21 +96,25 @@ impl DataProvider<LocationsV1Marker> for SourceDataProvider {
                 .main
                 .value
                 .localedisplaynames
-                .regions
+                .regions;
+            regions
                 .iter()
-                .filter(|&(k, _)| {
-                    /// Substring used to denote alternative region names data variants for a given region. For example: "BA-alt-short", "TL-alt-variant".
-                    const ALT_SUBSTRING: &str = "-alt-";
-                    /// Substring used to denote short region display names data variants for a given region. For example: "BA-alt-short".
-                    const SHORT_SUBSTRING: &str = "-alt-short";
-                    !k.contains(ALT_SUBSTRING) && !k.contains(SHORT_SUBSTRING)
-                })
                 .filter_map(|(region, value)| {
                     Some((
                         icu::locale::subtags::Region::try_from_str(region).ok()?,
                         value.as_str(),
                     ))
                 })
+                // Overwrite with short names, as we want to use those
```

Comment:
[UTS-35](https://www.unicode.org/reports/tr35/tr35-dates.html#Contents:~:text=From%20the%20TZDB%20get%20the%20country%20code%20for%20the%20zone%2C%20and%20determine%20whether%20there%20is%20only%20one%20timezone%20in%20the%20country.%20If%20there%20is%20only%20one%20timezone%20or%20if%20the%20zone%20id%20is%20in%20the%20%3CprimaryZones%3E%20list%2C%20format%20the%20country%20name%20with%20the%20regionFormat%2C%20and%20return%20it.) does not specify which names to use. I think `UK Time` is better than `United Kingdom Time`, because it is shorter.